Tilers/IfcTiler/README.md

```diff
@@ -0,0 +1,106 @@
+### See also
+ - [Installation notes](Install.md)
+ - [Developer's/Design notes](DesignNotes.md)
+
+
+### Intallation note : 
```

Comment:
This will become a library. Hence place the requirements in the setup.py ad-hoc section.
Note: since ifcOpenShell library is not on Pypi, use the git repo syntax in that requirement section.

Reply:
This will provoke errors when imported as [a py3dtiles-tilers library](https://github.com/VCityTeam/py3dtiles-tilers) anyhow. Still [opened an issue](https://github.com/VCityTeam/py3dtiles-tilers/issues/3) to clean up the Readme.md accordingly.